CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- BUGFIX: Updated MainTable so that the table headers aren't squashed when `expanding` is `true`.
```

Comment:
This seems to be from the other PR.

Reply:
Yeah I just added it in this PR while I was updating the changelog.

Reply:
But we don't have this bugfix in yet? The one that was merged was reverted, wasn't it?

Reply:
Oh yes, sorry, you're totally right! Removed.